package.json

```diff
@@ -118,7 +118,7 @@
     "cbioportal-frontend-commons": "0.1.0",
     "chart.js": "^2.6.0",
     "classnames": "^2.2.5",
-    "clinical-timeline": "0.0.19",
+    "clinical-timeline": "0.0.20",
```

Comment:
Presumably this will get updated. The bulk of the work is in that repo, so we need to wait for that to land before this PR can be reviewed and merged.

Link to PR: https://github.com/cBioPortal/clinical-timeline/pull/118